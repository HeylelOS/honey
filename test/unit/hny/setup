#!/bin/sh

set -e

"${TEST_BIN}" setup 'arxiv'

