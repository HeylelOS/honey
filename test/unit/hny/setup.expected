setup: Starting for 'arxiv'
Test Archive - Setup
